scripts/hal_alignment/hal_gene_liftover.py

```diff
@@ -83,21 +88,28 @@ def parse_region(region: str) -> SimpleRegion:
 
     """
     seq_region_regex = re.compile(
-        '^(?P<chrom>[^:]+):(?P<start>[0-9]+)-(?P<end>[0-9]+):(?P<strand>1|-1)$'
+        r'^(?P<chrom>[^:]+):(?P<start>[0-9]+)-(?P<end>[0-9]+):(?P<strand>.+)$'
     )
     match = seq_region_regex.match(region)
 
     try:
         region_chrom = match['chrom']  # type: ignore
-        match_start = match['start']  # type: ignore
-        match_end = match['end']  # type: ignore
+        match_start = int(match['start'])  # type: ignore
+        region_end = int(match['end'])  # type: ignore
```

Comment:
I guess `# type: ignore` is not required any more?
```suggestion
        match_start = int(match['start'])
        region_end = int(match['end'])
```

Reply:
Unfortunately `mypy` still produces an error message if this comment is not present.